pandas/io/formats/format.py

```diff
@@ -1482,6 +1482,11 @@ def _format_strings(self) -> List[str]:
         if is_categorical_dtype(values.dtype):
             # Categorical is special for now, so that we can preserve tzinfo
             array = values._internal_get_values()
+        elif values.dtype.kind == "O":
```

Comment:
prefer to use is_object_dtype

Reply:
i didnt check whether that works on JSONArray, just that `values.dtype == object` doesnt